galley/docker/Dockerfile.galley

```diff
@@ -4,13 +4,20 @@ ARG BASE_DISTRIBUTION=default
 # The following section is used as base image if BASE_DISTRIBUTION=default
 # hadolint ignore=DL3006
 FROM istionightly/base_debug as default
+COPY galley /usr/local/bin/
+
+# 1001 is chosen as the UID to be added to the istio group as this is the UID that galley in the docker container runs as.
+RUN addgroup --system --gid 1001 istio && \
```

Comment:
Please don't do this !!!

We can add it in the base image - but not in each and every docker image.

Reply:
some images are built from scratch instead of `base_debug`, how to add it in the base image?